playwright/element-web-test.ts

```diff
@@ -158,8 +160,13 @@ export const test = base.extend<
 
                 // Ensure the language is set to a consistent value
                 window.localStorage.setItem("mx_local_settings", '{"language":"en"}');
+
+                // Enable the lab features
+                for (const feature of enableLabFeatures) {
+                    window.localStorage.setItem(`mx_labs_feature_${feature}`, "true");
+                }
```

Comment:
Why not set these via the config.json override instead?

Reply:
Is there a difference?